source/kraken/worker.cpp

```diff
@@ -239,7 +241,19 @@ pbnavitia::Response Worker::next_stop_times(const pbnavitia::NextStopTimeRequest
                     type::AccessibiliteParams(), *data, false, request.count(),
                     request.start_page(), request.show_codes());
         case pbnavitia::NEXT_ARRIVALS:
-            return timetables::next_arrivals(request.arrival_filter(),
+            return timetables::previous_arrivals(request.arrival_filter(),
+                    forbidden_uri, until_datetime,
+                    request.duration(), request.nb_stoptimes(), request.depth(),
+                    type::AccessibiliteParams(),
+                    *data, false, request.count(), request.start_page(), request.show_codes());
```

Comment:
for `NEXT_ARRIVAL`, you call `timetables::previous_arrivals`? Need at least a comment...